### ISSUE TITLE
CentOS 7 not starting secondary nic

### ISSUE THREAD

**Issue by genebean** [mitchellh/vagrant#8229]
Vagrant 1.9.1
macOS 10.12.3
CentOS 7 with all patches

### Vagrantfile
```ruby
# -*- mode: ruby -*-
# vi: set ft=ruby :

# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
VAGRANTFILE_API_VERSION = "2"

Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
  config.vm.box = "genebean/centos-7-puppet-agent"

  if Vagrant.has_plugin?("vagrant-cachier")
    # Configure cached packages to be shared between instances of the same base box.
    # More info on http://fgrehm.viewdocs.io/vagrant-cachier/usage
    config.cache.scope = :box
  end

  config.vm.define "client" do |client|
    client.vm.hostname = "client.localdomain"
    client.vm.network "private_network", ip: "172.28.128.23"
  end

  config.vm.provider "virtualbox" do |v|
    v.gui = false
  end

end
```

### Debug output
https://gist.github.com/genebean/027d085362d77d1d3aafdea7aae0185b

### Expected behavior
Both network interfaces should be up on boot

### Actual behavior
private network is not up until running `systemctl restart network`

### Steps to reproduce
1. `vagrant up`
2. `vagrant ssh`
3. `ifconfig` -- note only two interfaces are up
4. `sudo systemctl restart network`
5. `ifconfig` -- note three interfaces are up

**Comment by kikitux** [comment]
duplicate of:
https://github.com/mitchellh/vagrant/issues/8096

Should be fixed on 1.9.2

**Comment by genebean** [comment]
@kikitux - Thanks. Any idea when 1.9.2 will be out?
